app/views/directives/centreGender.html

```diff
@@ -39,10 +39,17 @@
       <td ng-bind="prebookingreceived"></td>
       <td ng-bind="contingency"></td>
     </tr>
-    <tr ng-if="expectedin !== undefined">
-      <td>Expected incoming</td>
+
+    <tr ng-if="expectedin !== undefined" ng-click="showIncoming = expectedin.length>0 && !showIncoming"
+        ng-class="expectedin.length>0 ? showIncoming ? 'close' : 'open' : ''">
+      <td>
```

Comment:
I would suggest removing expectedin.length>0 in both the ng-click and the ng-class ; as I would suggest it is safe to presume that if expectedin is not undefined then expectedin.length>0